packages/ts/package.json

```diff
@@ -5,6 +5,9 @@
   "main": "index.ts",
   "module": "index.ts",
   "types": "index.ts",
+  "files": [
+    "README.md"
+  ],
```

Comment:
@saihaj this change broke the published npm package `@graphprotocol/graph-ts`, now it's empty: https://www.npmjs.com/package/@graphprotocol/graph-ts?activeTab=explore

Reply:
https://github.com/graphprotocol/graph-tooling/pull/1057

Reply:
Should be fixed here https://github.com/graphprotocol/graph-tooling/releases/tag/release-1675325398535